packages/core/src/createMaps.ts

```diff
@@ -101,6 +101,10 @@ export const createValueMap = (overrides?) =>
             o: getOverflows(),
             ox: getOverflows(),
             oy: getOverflows(),
+            oa: {
```

Comment:
`a: 'auto'` is missing. There short version of possible values so it can be minified in production builds.